crashtracker/src/rfc5_crash_info/stacktrace.rs

```diff
@@ -0,0 +1,178 @@
+// Copyright 2024-Present Datadog, Inc. https://www.datadoghq.com/
+// SPDX-License-Identifier: Apache-2.0
+
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use crate::NormalizedAddress;
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, JsonSchema)]
+pub struct StackTrace {
+    format: String,
+    frames: Vec<StackFrame>,
+}
+
+impl From<Vec<crate::StackFrame>> for StackTrace {
+    fn from(value: Vec<crate::StackFrame>) -> Self {
+        #[allow(clippy::type_complexity)]
+        fn convert_normalized_address(
+            value: Option<NormalizedAddress>,
+        ) -> (
+            Option<String>,      // build_id
+            Option<BuildIdType>, // build_id_type
+            Option<FileType>,    // file_type
+            Option<String>,      // path
+            Option<String>,      // relative_address
+        ) {
+            if let Some(normalized_address) = value {
+                let relative_address = Some(format!("{:#018x}", normalized_address.file_offset));
+                let (build_id, build_id_type, file_type, path) = match normalized_address.meta {
+                    crate::NormalizedAddressMeta::Apk(path_buf) => (
+                        None,
+                        None,
+                        Some(FileType::APK),
+                        Some(path_buf.to_string_lossy().to_string()),
+                    ),
+                    crate::NormalizedAddressMeta::Elf { path, build_id } => (
+                        byte_vec_as_hex(build_id),
+                        Some(BuildIdType::GNU),
+                        Some(FileType::ELF),
+                        Some(path.to_string_lossy().to_string()),
+                    ),
+                    crate::NormalizedAddressMeta::Pdb { path, guid, age } => (
+                        Some(format!(
+                            "{}:{age}",
```

Comment:
you can remove the ":".
From what I saw, Windbg will collect the guid and age and concatenate them together without a separator. The request to symbol server is done with the result.

Reply:
https://github.com/DataDog/libdatadog/issues/739